src/managed/OpenLiveWriter.Interop.Mshtml/mshtml/IHTMLChangeLog.cs

```diff
@@ -11,7 +11,7 @@ namespace mshtml
     public interface IHTMLChangeLog
     {
         [MethodImpl(MethodImplOptions.InternalCall, MethodCodeType = MethodCodeType.Runtime)]
-        void GetNextChange([In] ref byte pbBuffer, [In] int nBufferSize, out int pnRecordLength);
+        int GetNextChange([In] IntPtr pbBuffer, [In] int nBufferSize, out int pnRecordLength);
```

Comment:
@ScottIsAFool ah, this is a leftover from my hunt to diagnose the problem... It's not required for this PR, no, but it's a corrected signature. I can remove it.